### PR TITLE
[FEAT] Home - always display paragraph titles + add back to top

### PR DESCRIPTION
**Pull request by tbouffard**
Changes
- section titles sticked to the navbar when visible
- add a summary to navigate to various sections
- add a back to top button for a simplest navigation
- smooth scrolling
- card design cleaning:
  - light theme
  - the whole card is clickable

May be available live at https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/feat/always_display_title_in_home_add_back_to_top/examples/index.html

### CODE REVIEW

**Comment by tbouffard** [comment]
**Work in progress bb63c822**

example_home_navigation_bb63c822

**Comment by tbouffard** [comment]
### New improvements: cfa23f9 

improvements:
- h2 sticky position: the background color of the h2 elements must still be reworked to better match the theme
- back to top: no more text-decoration, better positioning when reaching the bottom of the page (it still moves a little)

improvements_cfa23f9


### Attempt to let collapse some sections: 613dcba

**Comment by tbouffard** [comment]
### Improvements 6e71fc3


improvements_6e71fc3

**Comment by tbouffard** [comment]
Titles have been removed from preview images in #98, so last try for cards.

| image on top 3203fe3 (as before) | title on top 59f5b24 |
| -- | --- |
| 01_image_on_top | 02_title_on_top |

**Comment by csouchet** [comment]
I prefer the title on top 🙂 
Nice job 😉

**Comment by NathalieC** [comment]
I also prefer "Title on top" and agree we need to rework the CSS to highlight them, maybe a light background color.

**Comment by tbouffard** [comment]
After working on the topic with @aibcmars (thanks again), this is now ready for review

- Card title on top of the card (as everybody agree on this) and made more visible
- no more glitch after clicking on the card or usage the tabulation to access the card
- 'back to top' button: always stay at bottom and more visible on hover/focus

image

**Review by aibcmars — APPROVED** [review]
cool:)

**Review by csouchet — APPROVED** [review]

File: examples/index.html [hunk 1]
```diff
@@ -50,237 +151,256 @@ <h3>Directly access to <span class="text-normal text-italic">bpmn-visualization<
...
+                    </a>
                 </div>
             </div>
         </section>
     </div>
+
 </section>
+    <!-- inspired by https://moderncss.dev/pure-css-smooth-scroll-back-to-top/ -->
+    <div class="back-to-top-wrapper float-right">
+        <a href="#top" class="back-to-top-link" aria-label="Scroll to Top">🔝</a>
```
Comment (aibcmars) [inline examples/index.html:402]: we need to adjust it for Chrome display

**Comment by csouchet** [comment]
On MacOS + safari: 
image

File: examples/index.html (hunk 1 above)
Comment (tbouffard) [reply examples/index.html:402]: Same not nice icon on Chrome 87 Ubuntu 18

Comment (aibcmars) [reply examples/index.html:402]: is it ok like this? 
icon comeas from spectre which we already use
image

other available icons:
https://picturepan2.github.io/spectre/elements/icons.html

**Pull request merged**